7-compute-engine/app/jobs/lookup_book_details_job.rb

```diff
@@ -43,7 +40,8 @@ def perform book
 
       # List of relevant books
       volumes = results.items
-
+# [END lookup_books]
```

Comment:
Do the delimiter comments have to match their indentation? This is kind of awkward from a code standpoint.

Reply:
I don't believe so, let me add that extra spacing and stage to verify.